frontend/components/ActivityDetails/InstallDetails/VppInstallDetailsModal/VppInstallDetailsModal.tsx

```
@@ -154,8 +154,7 @@ export const getStatusMessage = ({
             <div>
               The host acknowledged the MDM command to install <b>{appName}</b>
               {!isMyDevicePage && <> on {formattedHost}</>}, but the install
-              hasn&apos;t been verified. Fleet marks as failed if the install
-              isn&apos;t verified within {formattedVerifyTimeout}.
+              took longer than {formattedVerifyTimeout}, so Fleet marked it as failed.
```

ESLint

[error] Insert `⏎·············`

(prettier/prettier)